File: hard/16_underscorify_substring/underscorify_substring.test.js
// import { underscorifySubstring } from './underscorify_substring.js';
const underscorifySubstring = require('./underscorify_substring.js');

describe('TestProgram', () => {
  it('Test Case 1', () => {
    expect(underscorifySubstring("testthis is a testtest to see if testestest it works", "test")).toEqual("_test_this is a _testtest_ to see if _testestest_ it works");
  });
});

File: hard/16_underscorify_substring/underscorify_substring_test.py
import unittest
from underscorify_substring import underscorifySubstring

class TestProgram(unittest.TestCase):
    def test_case_1(self):
        self.assertEqual(
                    underscorifySubstring("testthis is a testtest to see if testestest it works", "test"),
                    "_test_this is a _testtest_ to see if _testestest_ it works",
                )
